mcs/class/System.XML/Test/System.Xml/XmlUrlResolverTests.cs

```diff
@@ -97,6 +97,7 @@ public void NullAbsoluteUriWithCustomSchemedRelativeUri ()
 		}
 
 		[Test]
+		[Category ("StaticNotWorking")] // Can't find .dll files when bundled in .exe

```

Comment:
`StaticLinkedAotNotWorking` seems reasonable as for example this test case can be make to work